index.ts

```diff
@@ -1,8 +1,9 @@
 // This file is to be imported for client code using the network interface.
 import {
   PersistedQueryNetworkInterface,
+  addPersistedQueries,
 } from './src/network_interface/ApolloNetworkInterface';
 
 export {
-  PersistedQueryNetworkInterface,
```

Comment:
I think we'd still want to be exporting `PersistedQueryNetworkInterface` until we do a slightly larger release